apollo-federation/src/operation/mod.rs

```diff
@@ -164,11 +164,15 @@ impl Operation {
                 .any(|f| f.has_defer())
     }
 
-    pub(crate) fn without_defer(self) -> Self {
```

Comment:
there is a `with_normalized_defer` function right above this that has a `todo!()`. As part of this PR can you remove that todo and just return the `NormalizedDefer` struct and add a comment mentioning `FED-95`? I didn't want to do it in #5426 so we could keep all of the defer things together.